builder/builder.mk

```diff
@@ -845,6 +845,59 @@ release/promote-oss/dev-to-passed-ci:
 	}
 .PHONY: release/promote-oss/dev-to-passed-ci
 
+# should run on every PR once the builds have passed
+# this is less strong than "release/promote-oss/dev-to-passed-ci"
+release/promote-oss/pr-to-passed-ci:
+	@set -e; { \
+		commit=$$(git rev-parse HEAD) ;\
+		dev_version=$$(aws s3 cp s3://datawire-static-files/dev-builds/$$commit -) ;\
+		if [ -z "$$dev_version" ]; then \
+			printf "$(RED)==> found no dev version for $$commit in S3...$(END)\n" ;\
+			exit 1 ;\
+		fi ;\
+		printf "$(CYN)==> $(GRN)Promoting $(BLU)$$commit$(GRN) => $(BLU)$$dev_version$(GRN) in S3...$(END)\n" ;\
+		echo "$$dev_version" | aws s3 cp - s3://datawire-static-files/passed-pr/$$commit ;\
+	}
+.PHONY: release/promote-oss/dev-to-passed-ci
```

Comment:
This should be `pr-to-passed-ci`, right? 

Also, the Circle config looks to me like this'll only run on PRs to `master` or to `rel/vX.Y.Z`, right?

Reply:
whoops and whoops. fixed both things